teams/crate-maintainers.toml

```diff
@@ -25,3 +25,7 @@ orgs = ["rust-lang"]
 weight = -200
 name = "Crate maintainers"
 description = "Maintaining official rust-lang crates such as log, libc, cc, and more."
+zulip-stream = "t-libs/crates"
+
+[[zulip-groups]]
+name = "crate-maintainers"
```

Comment:
The naming on convention we use on Zulip would have this as `T-crate-maintainers`.

Reply:
Updated.